src/Components/Header/Header.tsx

```diff
@@ -8,7 +8,7 @@ import './style.scss';
 
 interface Props {
   onValueChange?: (val: Imovie) => void;
-  selectedMovie: Imovie | null;
+  selectedMovie: Imovie | undefined;
```

Comment:
I think, it could be just optional property

Reply:
Thanks for the tip.